src/gui/guiFormSpecMenu.cpp

```diff
@@ -3951,14 +4048,8 @@ enum ButtonEventType : u8
 
 bool GUIFormSpecMenu::OnEvent(const SEvent& event)
 {
-	if (event.EventType==EET_KEY_INPUT_EVENT) {
+	if (event.EventType == EET_KEY_INPUT_EVENT) {
 		KeyPress kp(event.KeyInput);
-		if (event.KeyInput.PressedDown && (
-				(kp == EscapeKey) || (kp == CancelKey) ||
-				((m_client != NULL) && (kp == getKeySetting("keymap_inventory"))))) {
-			tryClose();
-			return true;
-		}
 
 		if (m_client != NULL && event.KeyInput.PressedDown &&
```

Comment:
There are two "if F12 then screenshot" stuff. If you take screenshot with this version, you get screenshot twice.
This one should be removed and only the other "if F12 then screenshot" below should be kept.